server/api/smallgroup/index.js

```diff
@@ -12,8 +12,8 @@ router.get('/', auth.hasRole('mentor'), controller.index);
 router.post('/', auth.hasRole('mentor'), controller.create);
 
 // Generate a daycode or return the current day code for your smallgroup
-router.post('/daycode', auth.hasRole('mentor'), controller.daycode);
-router.delete('/day/:dayCode', auth.hasRole('mentor'), controller.deleteDay);
+router.post('/:id/daycode', auth.hasRole('mentor'), controller.daycode);
```

Comment:
If you do by code you will delete all randomly generated attendance codes in the db. To stay consistent I'm pulling from just daycodes from the small group